src/ResourceManager/MachineLearningCompute/Commands.MachineLearningCompute.Test/ScenarioTests/MLCTests.ps1

```diff
@@ -70,6 +66,15 @@ function TeardownTest([String] $ResourceGroupName)
     Write-Debug "Delete resource group"
     Write-Debug " Resource Group Name : $resourceGroupName"
     Remove-AzureRmResourceGroup -Name $ResourceGroupName -Force
+
+	foreach ($g in Find-AzureRmResourceGroup)
```

Comment:
@shutchings `Get-AzureRmResourceGroup`

Reply:
Couldn't figure out how to get this to work. Instead I'm getting the resource group a different way.